index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Basic Calculator</title>
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="script.js"></script>
+</head>
+<body>
+    <h1>Web Calculator</h1>
+    <h2>Enter Two Numbers To Calculate</h2>
+    <input class="box" type="number" name="firstBox" id="number">
+<br />
+    <input class="box" type="number" name="secondBox" id="number1">
+<br />
+    <input class="box" type="number" name="thirdBox" id="result" readonly>
```

Comment:
Don't use an input element for displaying the answer. It is great that you made it readonly, but it is better to use a simple span, which you can style how you like. The idea is to only use "input" elements to actually receive input.